buddypress.org/public_html/wp-content/themes/bb-base/bbpress/form-topic-split.php

```
@@ -39,7 +39,7 @@
 							<div>
 								<input name="bbp_topic_split_option" id="bbp_topic_split_option_reply" type="radio" checked="checked" value="reply" />
 								<label for="bbp_topic_split_option_reply"><?php printf( __( 'New topic in <strong>%s</strong> titled:', 'bbpress' ), bbp_get_forum_title( bbp_get_topic_forum_id( bbp_get_topic_id() ) ) ); ?></label>
-								<input type="text" id="bbp_topic_split_destination_title" value="<?php printf( __( 'Split: %s', 'bbpress' ), bbp_get_topic_title() ); ?>" size="35" name="bbp_topic_split_destination_title" />
+								<input type="text" id="bbp_topic_split_destination_title" value="<?php echo esc_attr( sprintf( __( 'Split: %s', 'bbpress' ), bbp_get_topic_title() ) ); ?>" size="35" name="bbp_topic_split_destination_title" />
```

PHPStan (2.2.8)

[error] Function __ invoked with 2 parameters, 1 required.


(arguments.count)